### DISCUSSION TITLE
Remove tabs in container

### DISCUSSION THREAD

**Discussion by cuong-huynh-tamara** [squidfunk/mkdocs-material#3741] [Q&A] [answered]
Hi,

Currently, I'm customizing the document as as per design, I can add Tabs to header by override header.html but the problem is Tabs is still show in container, so it will be duplicated with header, do we have a solution to remove Tabs in main container? I also tried to override base.html but it was breaking everything.

Thanks

**Comment by squidfunk** [comment, marked as answer]
I think the behaviour you’re looking for is implemented as what we call [sticky navigation tabs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#sticky-navigation-tabs). The tabs will then only be added to the header, not to the main container.

**Reply by cuong-huynh-tamara** [reply]
Thanks a lot, It's working now.
